x-pack/plugins/cases/public/components/recent_cases/index.tsx

```diff
@@ -0,0 +1,89 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+
+import { EuiFlexGroup, EuiFlexItem, EuiHorizontalRule, EuiText, EuiTitle } from '@elastic/eui';
+import React, { useMemo, useState } from 'react';
+
+import * as i18n from './translations';
+import { CaseDetailsHrefSchema } from '../links';
+import { RecentCasesFilters } from './filters';
+import { RecentCasesComp } from './recent_cases';
+import { FilterMode as RecentCasesFilterMode } from './types';
+import { useCurrentUser } from '../../common/lib/kibana';
+
+export interface RecentCasesProps {
+  allCasesLink: JSX.Element;
+  createCaseHref: string;
+  getCaseDetailsHref: (caseDetails: CaseDetailsHrefSchema) => string;
+  onCaseDetailsNavClick: (caseDetails: CaseDetailsHrefSchema) => void;
+  perPage: number;
+}
+
+const RecentCases = ({
```

Comment:
<img width="337" alt="Screen Shot 2021-03-30 at 7 16 51 AM" src="https://user-images.githubusercontent.com/6935300/112994898-f1b73580-9127-11eb-9cba-9ed5492c4315.png">
this is the Cases widget on the overview page.